kong/plugins/ip-restriction/schema.lua

```diff
@@ -8,8 +8,8 @@ return {
     { config = {
         type = "record",
         fields = {
-          { whitelist = { type = "array", elements = typedefs.cidr_v4, }, },
-          { blacklist = { type = "array", elements = typedefs.cidr_v4, }, },
+          { whitelist = { type = "array", elements = typedefs.cidr_v4_v6, }, },
```

Comment:
Here as I commented above, you could just use:
```
typedefs.ip_or_cidr
```

Reply:
we can break existing third party plugins using `validate_ip_or_cidr` (look at my comment above)

Reply:
no we cannot as WE ARE NOT CHANGING typedef. Only the schema of ip-restriction plugin.